jax/core.py

```diff
@@ -32,6 +32,7 @@
 from .config import FLAGS, config
 from . import linear_util as lu
 from . import source_info_util
+from .numpy import _operations, _nondiff_methods, diff_methods
```

Comment:
We don't want a dependence on jax.numpy in core (I think it'll lead to circular import issues, and to the extent that we want some kind of dependence tree rather than every module importing every other module, we want core to be minimal).

Maybe this was stale code anyway?

```suggestion
```